ui/app/components/mount-backend-form.js

```diff
@@ -116,7 +116,7 @@ export default Component.extend({
       return;
     }
     this.get('flashMessages').success(
-      `Successfully mounted ${type} ${this.get('mountType')} method at ${path}.`
+      `Successfully mounted the ${type} ${this.get('mountType')} engine at ${path}.`
```

Comment:
I think we still want it to say method for auth & engine for secrets? @meirish?

Reply:
Yeah it looks like it was wrong before, but +1 to engine if mountType is secret and method if mountType is auth.

Reply:
nice catch, thank you!